### DISCUSSION TITLE
Hungarian Layout/Keymap

### DISCUSSION THREAD

**Discussion by h0wReYa** [monkeytypegame/monkeytype#893] [Ideas] [answered]
I can use hungarian language, but i can't use hungarian layout. In this way i can't type, i think we need layout option like this:
[Hungarian keyboard layout](https://upload.wikimedia.org/wikipedia/commons/thumb/a/a8/KB_Hungary.svg/1920px-KB_Hungary.svg.png)
Thanks

**Comment by Miodec** [comment, marked as answer]
Layout override doesnt support alt gr layer, and it looks like this layout depends on it. Why cant you add hungarian layout in your system?

**Reply by h0wReYa** [reply]
monkeytype language: hungarian
layout override: qwertz
winsows 10: HUN

If I type "é", which is next to "l" (L not i) monkeytype reflects as I type "ö" , which should be next to "9"
With this settings my "é" is "ö", my "á" is "ä", and i think this is layout problem.

Maybe this can help:

hungarian: {
    keymapShowTopRow: false,
    iso: true,
    keys: [
        "0§", "1'", "2\"", "3+", "4!", "5%", "6/", "7=", "8(", "9)", "öÖ", "üÜ", "óÓ",
        "qQ", "wW", "eE", "rR", "tT", "zZ", "uU", "iI", "oO", "pP", "őŐ", "úÚ", "",
        "aA", "sS", "dD", "fF", "gG", "hH", "jJ", "kK", "lL", "éÉ", "áÁ", "űŰ",
        "íÍ", "yY", "xX", "cC", "vV", "bB", "nN", "mM", ",?", ".:", "-_",
        " "
    ]
},

**Reply by Miodec** [reply]
Try using `default` layout override.
